hypertrace-gradle-docker-java-application-plugin/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+
+## [0.2.4]
+### Changed
+- Default image has changed from gcr.io/distroless/java:11 to gcr.io/distroless/java:11-debug
+
+## [0.2.5]
```

Comment:
The new version I see is '0.3.0'.

Reply:
Yeah that was a mistake. I've fixed it locally, was waiting to see if more changed were needed otherwise I'll update the docs today.